packages/squiggle-lang/src/js/index.ts

```diff
@@ -117,6 +117,8 @@ function createTsExport(
       );
     case "EvBool":
       return tag("boolean", x.value);
+    case "EvCall":
+      return tag("call", x.value);
```

Comment:
you also need `tagged<"call", string>;` about line 96.

Reply:
Done